src/Tooling/Bit.Tooling.Templates/TodoTemplate/Api/wwwroot/swagger/swagger-utils.js

```diff
@@ -14,17 +17,44 @@ const initLoginForm = () => {
 
     overrideSwaggerAuthorizeEvent(swagger);
     overrideSwaggerLogoutEvent(swagger);
+    tryAuthorizeWithLocalData(swagger);
     showLoginUI(swagger);
 }
 
+const tryAuthorizeWithLocalData = (swagger) => {
+    if (isAuthorized(swagger))
+        return;
+
+    const token = getCookie(AUTHORIZATION_TOKEN_KEY);
+    if (!token)
+        return;
+
+    const authorizationObject = getAuthorizationRequestObject(token);
+    swagger.authActions.authorize(authorizationObject);
+}
+
 const overrideSwaggerAuthorizeEvent = (swagger) => {
     const auth = swagger.authActions.authorize;
-    swagger.authActions.authorize = (args) => attachReloadPageToEvent(swagger, auth, args);
+    swagger.authActions.authorize = async (args) => {
+        const result = await auth(args);
+
+        if (!isCookieSet(AUTHORIZATION_TOKEN_KEY)) {
+            setCookie(AUTHORIZATION_TOKEN_KEY, result.payload.bearerAuth.value, accessTokenAge);
+        }
+
+        reloadPage(swagger);
+        return result;
+    };
 }
 
 const overrideSwaggerLogoutEvent = (swagger) => {
     const logout = swagger.authActions.logout;
-    swagger.authActions.logout = (args) => attachReloadPageToEvent(swagger, logout, args);
+    swagger.authActions.logout = async (args) => {
+        const result = await logout(args);
```

Comment:
plz remove the `logout` variable and use its value directly here.

Reply:
@milad-karimifard is trying to override logout functionality here, and to preserve original functionality, he has to store original function somewhere

I'd recommend something like `originalLogoutImpl`

Reply:
tnx @ysmoradi 
rename this `logout` variable to `originalLogout` please.

Reply:
@msynk sure, I did that